src/ui/triggers_main_area.ui

```diff
@@ -538,7 +538,7 @@
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
-            <string comment="This is the tooltip for the QLineEdit that shows - but does not permit changing - the sound file used for a trigger." extracomment="Please ensure that the text used for &quot;Choose file ,,,&quot; here is the same as is used for the button that it refers to.">&lt;p&gt;This displays the file selected to play when the trigger fires. To change this used the &lt;i&gt;Choose file...&lt;/i&gt; button or to remove it use the clear button.&lt;/p&gt;</string>
+            <string comment="This is the tooltip for the QLineEdit that shows - but does not permit changing - the sound file used for a trigger.">&lt;p&gt;Sound file to play when the trigger fires.&lt;/p&gt;</string>
```

Comment:
This will ***never*** be seen as it will get over-written by the file name - even if that is an empty `QString` for the first trigger that is examined...

Reply:
![image](https://user-images.githubusercontent.com/110988/61585958-5788a700-ab69-11e9-8669-6bd7e845083c.png)

Reply:
So that is the wrong text. Expected was `Sound file to play when the trigger fires`

Reply:
Good point, it looks like it is showing the tooltip for the _overall_ area instead. We have several overlapping tooltips.